package/gnu/project/src/docker/create/default/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile will install flow123d into a flow123d/install image
 
-FROM flow123d/install-gnu
+FROM flow123d/install-gnu:${images_version}
```

Comment:
Jinde se ve FROM myslim verze neuvadi to znamena ze to bere asi latest a nemusi to byt dobre. Asi bude vhodne opravit vsude.